definitions/checks/backup/incremental_parent_type.rb

```diff
@@ -0,0 +1,31 @@
+module Checks::Backup
+  class IncrementalParentType < ForemanMaintain::Check
+    metadata do
+      description 'Check if the incremental backup has the right type'
+      tags :backup
+      param :incremental_dir, 'Path to existing backup directory'
+      param :online_backup, 'Select for online backup', :flag => true, :default => false
+      param :sql_tar, 'Will backup include PostgreSQL tarball', :flag => true, :default => false
+      manual_detection
+    end
+
+    def run
+      return unless @incremental_dir
+
+      backup = ForemanMaintain::Utils::Backup.new(@incremental_dir)
+
+      existing_type = backup.online_backup? ? 'online' : 'offline'
```

Comment:
How much are the words online and offline used in the code? This feels like a spot that could be prone to typos and failures - consider defining a constant somewhere that can be used.

Reply:
In which code? this check? or all of FM?

Reply:
I was thinking about the latter.